ui/css/treeherder-job-buttons.css

```diff
@@ -12,6 +12,7 @@
   vertical-align: 0;
   line-height: 1.32;
   display: none;
+  transition: all 0.05s;
```

Comment:
This one I'm not so sure of, when I'm rapidly changing selected jobs via keyboard shortcuts (n,p or left,right) it seems more difficult to see what is happening, than when no transition exists? It looks ok on manual selection, but I'm not sure of its overall UX gain. I think you'd want to check with Sheriffs on this one, before adding it.

I also hit some strange states which I think were related to this transition. The entire job block would render disabled, with that rapid keyboard shortcut selection workflow. Usually by about the 2nd or 3rd push.

![rapidleftrightnpnavigation](https://cloud.githubusercontent.com/assets/3660661/9969431/777597ba-5e1c-11e5-9b80-1c4defacc5b6.jpg)

My vote would be to omit this change for now, but we could land the group expand/collapse transition :)